doc/api/cli.md

```diff
@@ -2868,7 +2868,10 @@ The following values are valid for `mode`:
 ### `--use-system-ca`
 
 Node.js uses the trusted CA certificates present in the system store along with
-the `--use-bundled-ca`, `--use-openssl-ca` options.
+the `--use-bundled-ca` option and the `NODE_EXTRA_CA_CERTS` environment variable.
+On platform other than Windows and macOS, this loads certificates from the directory
+and file trusted by OpenSSL, similar to `--use-openssl-ca`, with the difference being
+that it caches the certificates after first load.
 
 This option is only supported on Windows and macOS, and the certificate trust policy
```

Comment:
This is no longer true with this PR, or does the trust policy statement only apply on Windows/macOS?